themes/digital.gov/layouts/shortcodes/img-right.html

```diff
@@ -1,3 +1,12 @@
+{{/* Renders image on right side of screen
+
+  ** Note for best practices, can also be found in the WIKI with short code** 
+  Short code should be added above content blocks within files to ensure alignments are correct 
+
+  <!-- TODO: move lines 11-20 into comment block for better break down of use -->
```

Comment:
This is unclear.

Reply:
I have a follow up ticket to address this better and add more documentation to the file